client/src/components/SaveButton.js

```diff
@@ -1,38 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom/';
 import PropTypes from 'prop-types';
-import { Button, Icon } from 'antd';
+import { Button, Icon, Popover } from 'antd';
 
 function SaveButton(props) {
   const { authed, isSaved, deleteResourceHandler, saveResourceHandler } = props;
 
+  const errorContent = (
+    <div>
+      You must be logged in to use this feature! Login{' '}
+      <Link to="/login">here</Link>.
+    </div>

```

Comment:
This can be wrapped in a fragment (<> and </>) rather than a div (unless that causes weird issues, idk)

Reply:
changed!